### DISCUSSION TITLE
Relating compute_paths CIRs to Scene's coverage_map in Sionna RT

### DISCUSSION THREAD

**Discussion by ifelse1984** [NVlabs/sionna#270] [Q&A]
For Sionna RT, the CIRs and Coverage Maps are computed with different functions and classes. Is there a way to reconcile the two?

For instance, given a Tx (Array) + Rx pair, I can use compute path to get the CIR. Using a single Tx (Array), I can compute the coverage map, for which I can identify one of the cell centers that is closest to my Rx. Can I obtain the same path gain value (which seems time-invariant) on the coverage map from the CIR (which is time-variant)?

**Comment by jhoydis** [comment]
In short, the answer is no. 

Coverage maps are computed as described [here](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.coverage_map). They correspond to the normalized integral of the incoming energy across small surface patches. An approximation of this would be to compute the CIRs at several positions within a small region and compute their average energy. That means that you cannot perfectly recover the same path gain from a single CIR. However, they should be not too different if the cell size of the coverage map is not too large.

Why would you need to do this?

**Reply by ifelse1984** [reply]
Thank you for the details!

With regards to why:
1. If I were interested in only a few candidate positions/certain regions of the map, and I already have computed the CIRs, I was thinking of whether I have enough information to compute the coverage map (or an approximation of it) for that area without needing to re-run another ray-tracing routine.
2. More broadly, I am also hoping to understand better the relation between the two routines, and how they are related to each other despite being implemented by different methods (related to my follow-up question is below).

One thing that confuses me is that the channel coefficient h (in [EM primer](https://nvlabs.github.io/sionna/em_primer.html#equation-h-b)) is time varying, but h used in the coverage map computation (in the [coverage map description](https://nvlabs.github.io/sionna/api/rt.html#equation-cm-def)) depends on position, but seems to be independent of time. Hence, I was wondering how do I reconcile the two, even in the limit where the cell-size is very small?
